.github/workflows/checks-integration.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - master
+  # TODO(c-bata): Revert this temporary change after checking this PR.
+  pull_request: {}
```

Comment:
```suggestion
```

Reply:
It seems to have been resolved.

```
Run mypy . --warn-unused-ignores
optuna/integration/pytorch_distributed.py:137: error: Incompatible types in assignment (expression has type "Optional[ProcessGroup]", variable has type "ProcessGroup")  [assignment]
optuna/integration/pytorch_lightning.py:98: error: "Trainer" has no attribute "training_type_plugin"  [attr-defined]
optuna/integration/allennlp/_pruner.py:26: error: Module "allennlp.training" does not explicitly export attribute "GradientDescentTrainer"  [attr-defined]
optuna/integration/allennlp/_pruner.py:27: error: Module "allennlp.training" does not explicitly export attribute "TrainerCallback"  [attr-defined]
tests/integration_tests/test_chainer.py:23: error: Cannot assign to a type  [misc]
tests/integration_tests/test_chainer.py:23: error: Incompatible types in assignment (expression has type "Type[object]", variable has type "Type[DatasetMixin]")  [assignment]
tests/integration_tests/allennlp_tests/test_allennlp.py:303: error: "Module allennlp.training" does not explicitly export attribute "GradientDescentTrainer"  [attr-defined]
Found 7 errors in 5 files (checked 302 source files)
Error: Process completed with exit code 1.
```

https://github.com/optuna/optuna/actions/runs/4130496890/jobs/7137247993